docs: ui.contextual_help (#974)
deephaven/deephaven-plugins: plugins/ui/docs/components/contextual_help.md

# Contextual Help

Contextual help can be used to show extra information about the state of a component.

## Example

For the contextual help component, both the `heading` and `content` props are required. 

```python
from deephaven import ui

my_contextual_help_basic = ui.contextual_help(
    heading="Need Help",
    content="If you are having issues accessing your account, contact our customer support team for help.",
    variant="info",
)
```

## Placement

> The contextual help component supports different placement options for when the positioning of the popover needs to customized.

Reviewer: ```suggestion
The contextual help component supports different placement options for when the positioning of the popover needs to be customized.
```

> The contextual help component supports different placement options for when the positioning of the popover needs to be customized.

Reviewer: ```suggestion
The contextual help component supports different placement options for when the popover's positioning needs to be customized.
```

```python
from deephaven import ui

@ui.component
def ui_contextual_help_placement_examples():
    return [
        ui.contextual_help(
            heading="Need Help",
            content="If you are having issues accessing your account, contact our customer support team for help.",
            variant="info",
        ),
        ui.contextual_help(
            heading="Need Help",
            content="If you are having issues accessing your account, contact our customer support team for help.",
            variant="info",
            placement="top start",
        ),
        ui.contextual_help(
            heading="Need Help",
            content="If you are having issues accessing your account, contact our customer support team for help.",
            variant="info",
            placement="end",
        ),
    ]

my_contextual_help_placement_examples = ui_contextual_help_placement_examples()
```

## Events

> The contextual help componenthas an `on_open_change` prop, triggered when the the popover opens or closes.

Reviewer: ```suggestion
The `on_open_change` prop is triggered when the popover opens or closes.
```

or at least: The contextual help component has an `on_open_change` prop, triggered when the popover opens or closes.

# Contextual Help

> The contextual help feature shows extra information about the state of a component, or a total view.

Author: I had meant view as in an intentional a grouping of components, sort of related to ui.view. But yeah, getting rid of this, not super helpful.
